My Personal Website - HTML only

PhilippAubert opened the pull request:
## please feel free to take a look 

### comments are welcome :blush:

PhilippAubert committed:
I add a DS.STORE to .gitignore

PhilippAubert committed:
Add first content

PhilippAubert committed:
Here's my first draft of a 'personal site'

PhilippAubert committed:
committed as a new version

PhilippAubert committed:
Merge pull request #1 from PhilippAubert/markup_assign

Philipp Aubert - About Me Website 1st Draft

johdyn commented:
LGTM

namirsab commented on index.html:
```diff
@@ -0,0 +1,166 @@
+        
+            </nav>
+            
+            <figure> 
```
Amazing, using `figure`! very good!

PhilippAubert replied:
Thank you :blush:

namirsab commented on index.html:
```diff
@@ -0,0 +1,166 @@
+
+          <article> 
+            <h2>Abstract Painting</h2>
+            <p>Challening space with the line</p>
```
I think you want to write here `Challenging`?

PhilippAubert replied:
I did. What a blunder :satisfied:

namirsab commented on markup-assign.html:
```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html>
```
You have 2 files with the same content. Is this intended?

PhilippAubert replied:
I still do. Of course, index.html is the proper one. 

This happened before I did the git push - I removed the file and even deleted the branch, but it didn't update so I must have missed something at one point. I'll take care of that right now. :blush:

mpagels reviewed (approved):
What does the markup-assign.html do? Seems like a copy. If not needed you can delete it before merge, to avoid data "noise" in the project. ;-)

Very good Job Philipp!

namirsab commented on index.html:
```diff
@@ -0,0 +1,166 @@
+    <head>
+
+        <meta charset="utf-8">
+        <title>Assignment HTML</title>
```
Even though this is an assignemnt, think of it as your personal project. give it a title that you'd like to see in a the browser tab 😉

PhilippAubert replied:
You're absolutely correct. :satisfied:

namirsab reviewed (changes requested):
Hey @PhilippAubert really good job! I love the content, I had lots of fun reading it! (and i'll definetely check your soundcloud haha)

I've added some little requests, please take a look to them. Just some small details.

PhilippAubert committed:
Update index.html

PhilippAubert reviewed (commented):
Thanks for the input, Namir and Martin. You guys rock :blush:

PhilippAubert committed:
Delete markup-assign.html

As Martin has pointed out, two files might cause "data-noise". So one had to go :punch:
